drake/systems/LCMSystem.h

```diff
@@ -1,16 +1,16 @@
 #ifndef DRAKE_SYSTEMS_LCMSYSTEM_H_
 #define DRAKE_SYSTEMS_LCMSYSTEM_H_
 
-#include <unordered_map>
+#include <lcm/lcm-cpp.hpp>
 #include <mutex>
-#include <thread>
 #include <stdexcept>
-#include <lcm/lcm-cpp.hpp>
-#include "lcmtypes/drake/lcmt_drake_signal.hpp"
-#include "drake/systems/System.h"
+#include <thread>
+#include <unordered_map>
+#include "drake/drakeLCMSystem_export.h"
 #include "drake/systems/Simulation.h"
+#include "drake/systems/System.h"
 #include "drake/systems/cascade_system.h"
-#include "drake/drakeLCMSystem_export.h"
+#include "lcmtypes/drake/lcmt_drake_signal.hpp"
```

Comment:
These 10 lines of #include dancing shrapnel would be a good example to cite in #1977 of something we want to fix with a clang-format dotfile.